assets/src/edit-story/components/richText/customImport.js

```diff
@@ -40,7 +40,7 @@ function customInlineFn(element, { Style }) {
 function importHTML(html) {
   const htmlWithBreaks = (html || '')
     // Re-insert manual line-breaks for empty lines
-    .replace(/\n(?=\n)/g, '\n<br />')
+    .replace(/\n/g, '\n<br />')
```

Comment:
+1
![nls](https://user-images.githubusercontent.com/21036224/88858748-c8d94c00-d1f8-11ea-831b-3158b45306de.gif)

Reply:
This was my concern that the regex was there. Perhaps the solution here is to not trim trailing and leading line breaks rather than internal ones. Then we can unit test the function to verify that.

Reply:
This should be fixed now.